controller/sessions.js

```diff
@@ -0,0 +1,83 @@
+var
+  _ = require('lodash'),
+  changeCase = require('change-case'),
+  async = require('async'),
+  projects = require('./projects'),
+  inquirer = require('inquirer'),
+  colors = require('colors'),
+  cancelOption = '[cancel]';
+
+function listSessions(sessions) {
+  console.log("Grabbing sessions. . .".yellow);
+  const sessionNames = _.map(sessions, session =>
+    changeCase.titleCase(session.cohort)
+  );
+  const resolve = {
+    sessions,
+    sessionNames
+  };
+  return sessionNames;
+}
+
+module.exports.listSessions = listSessions;
+
+function pluckSession(cohort, sessions) {
+  const dashCohort = changeCase.paramCase(cohort);
+  const sessionsArray = _.map(sessions, session => session);
+  return sessionsArray.reduce(function(result, current) {
+    return current.cohort === dashCohort ? current : result;
+  });
+}
+
+module.exports.pluckSession = pluckSession;
+
+function selectSession(sessions) {
+  const parsedSessions = JSON.parse(sessions);
+  const sessionNames = listSessions(parsedSessions);
+  return new Promise(function (res, rej) {
+    async.waterfall([
+      function (next) {
+        inquirer.prompt(
+          [
+            {
+              type: 'list',
+              name: 'class',
+              message: `Select the class to ${projects.action} a project from`,
+              choices: sessionNames.concat(cancelOption)
+            }
+          ],
+          function (response) {
+            if (response.class === cancelOption) {
+              console.log('Installation cancelled, bye bye!'.green);
+              process.exit();
+            }
+            next(null, response);
+          }
+        );
+      },
+      function (response, next) {
+        inquirer.prompt(
+          [
+            {
+              type: 'confirm',
+              name: 'action',
+              message: `You selected ${response.class}: Is that correct?`,
```

Comment:
This template literal isn't going to work on c9 (Node 4). I can't remember if you're using [n](https://github.com/tj/n) or nvm to install a different Node version on c9. It's probably better to just use whatever version c9 is already using and target that version. Make sure you use the same version in development as in production to limit these discrepancies. 
If you want to use ES2015, use babel to transpile to ES5.